content/doc/developer/tutorial-improve/update-parent-pom.adoc

```diff
@@ -19,6 +19,52 @@ The most recent parent POM files also provide additional static analysis like sp
 // Create the branch
 include::doc/developer/tutorial-improve/_create-a-branch.adoc[]
 
+
+== Maven configuration
+
+Before updating the parent POM (Project Object Model) using the "mvn -ntp version:update-parent" command, you must make sure that the Maven configuration files are correctly configured.
+This is important because the plugin:maven-hpi-plugin[maven-hpi-plugin], used for developing Jenkins plugins, might require some additional global configurations to function correctly with older plugin environments.
```

Comment:
```suggestion
This is important because the link:https://github.com/jenkinsci/maven-hpi-plugin[maven-hpi-plugin], used for developing Jenkins plugins, requires some additional global configurations to function correctly with older plugin environments.
```
The plugin macro cannot be used here, as the maven-hpi-plugin is not a Jenkins plugins that can be found on plugins.jenkins.io. Therefore, it would need the link macro and have the link leading to the github page for the plugin.

Reply:
okay got it sir done